_data/accessibility-tests/footer.yml

```
@@ -0,0 +1,97 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: In the big footer variant, the email input field announces its purpose.
+    summary_additional: When you navigate to the email address input field in the big footer variant, you hear that the input expects the user's email address.
```

@amyleadem @amycole501 @alex-hull 
Looking at how we might include the `autocomplete` attribute here. I don't think we should lose the existing summary because it feels important to the success criteria. What do we think about just adding a line here like this:

```suggestion
  - summary: In the big footer variant, the email input field announces its purpose.
    summary_additional: When you navigate to the email address input field in the big footer variant, you hear that the input expects the user's email address. Additionally, the input field has the `autocomplete="email"` attribute.
```

---

I like this solution for now. I think we need to do the following for this item:
1. Add a sentence (like the suggestion above) about the autocomplete attribute.

---

I like this solution for now. In the future, I might suggest we break it up into separate checks with more information about how to check for the attribute in the code. 

I see the following next steps for this item:
1. Add a sentence (like the suggestion above) about the autocomplete attribute.
2. Mark this `test_status` value as "exception"
3. Include a `status_details` key. Possible suggestion for a summary: "The input is appropriately marked as an "email" input type, but does not have the autocomplete attribute in the default component example. Be sure to add the `autocomplete="email"` attribute to the email input element in your implementation of the component.
4. Point to issue uswds/uswds#6001 in the status summary

---

Great thinking here! Added in [ce3ac3f](https://github.com/uswds/uswds-site/pull/2760/commits/ce3ac3f7822affb617ed833ad5d1ac19d9bd02c4)